ios/FluentUI/Command Bar/MultilineCommandBar.swift

```diff
@@ -17,7 +17,7 @@ public class MultilineCommandBarRow: NSObject {
 }
 
 @objc(MSFMultilineCommandBar)
-public class MultilineCommandBar: BottomSheetController {
```

Comment:
This changes the public APIs for MultilineCommandBar (because now it no longer will expose the public APIs from BottomSheetController). Have you looked at usages of this in OMR/our partner repos to make sure this doesn't cause unintentional breaking changes or regressions?

Reply:
but this is a new control for the very first time. there won't be any public api regression.

Reply:
This also broke the pod lib lint so I've changed it to inherit from `UIViewController`

Reply:
Oh duh. Good call!